pkg/managed/utils_test.go

```diff
@@ -0,0 +1,220 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+package managed
+
+import (
+	asserts "github.com/stretchr/testify/assert"
+	cohoprclientset "github.com/verrazzano/verrazzano-coh-cluster-operator/pkg/client/clientset/versioned"
+	clientset "github.com/verrazzano/verrazzano-crd-generator/pkg/client/clientset/versioned"
+	clientsetfake "github.com/verrazzano/verrazzano-crd-generator/pkg/client/clientset/versioned/fake"
+	cohcluclientset "github.com/verrazzano/verrazzano-crd-generator/pkg/clientcoherence/clientset/versioned"
+	cohcluclientsetfake "github.com/verrazzano/verrazzano-crd-generator/pkg/clientcoherence/clientset/versioned/fake"
+	istioclientset "github.com/verrazzano/verrazzano-crd-generator/pkg/clientistio/clientset/versioned"
+	istioclientsetfake "github.com/verrazzano/verrazzano-crd-generator/pkg/clientistio/clientset/versioned/fake"
+	domclientset "github.com/verrazzano/verrazzano-crd-generator/pkg/clientwks/clientset/versioned"
+	domclientsetfake "github.com/verrazzano/verrazzano-crd-generator/pkg/clientwks/clientset/versioned/fake"
+	helidionclientset "github.com/verrazzano/verrazzano-helidon-app-operator/pkg/client/clientset/versioned"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutil"
+	wlsoprclientset "github.com/verrazzano/verrazzano-wko-operator/pkg/client/clientset/versioned"
+	istioauthclientset "istio.io/client-go/pkg/clientset/versioned"
+	istioauthclientsetfake "istio.io/client-go/pkg/clientset/versioned/fake"
+	extclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	extclientsetfake "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/fake"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/rest"
+	"os"
+	"testing"
+)
+
+// TestBuildManagedClusterConnection test the creation of a managed cluster connection
```

Comment:
tests the connection...

Reply:
Not really sure what you are saying here.  Do you want me to reword the doc?

Reply:
use the work tests not test